src/deepsparse/pipelines/text_generation.py

```diff
@@ -0,0 +1,547 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import Dict, List, Optional, Tuple, Type
+
+import numpy
+from onnx import ValueInfoProto
+from pydantic import BaseModel, Field
+from transformers import AutoConfig, AutoTokenizer
+
+from deepsparse import Pipeline
+from deepsparse.transformers.helpers import (
+    get_onnx_path_and_configs,
+    overwrite_transformer_onnx_model_inputs,
+)
+from deepsparse.transformers.pipelines import TransformersPipeline
+
+
+OPT_CACHE_HIDDEN_DIM = 64
```

Comment:
I could not find an elegant way to access this variable (not in the self.config variable sadly)